app/javascript/mastodon/service_worker/web_push_notifications.js

```diff
@@ -1,5 +1,6 @@
 import IntlMessageFormat from 'intl-messageformat';
 import locales from './web_push_locales';
+import { unescape } from 'lodash';
```

Comment:
Does this get tree-shaken or does it pull in the entire package?

Reply:
https://github.com/tootsuite/mastodon/blob/5ea643b27908b14bd89ff068fc87e446e8cbcd32/.babelrc#L38

[`babel-plugin-lodash`](https://github.com/lodash/babel-plugin-lodash) is used.